src/components/Pages/Assets/Tokens/TokensView.tsx

```diff
@@ -0,0 +1,36 @@
+import { memo } from 'react';
+
+import { useAssets } from '@hooks/useAssets';
+
+import Tokens from './Tokens';
+
+const TokensView = () => {
+  const {
+    getNativeBalance,
+    getNativeMetadata,
+    getTokensBalances,
+    getTokensMetadata,
+    nativeBalance,
+    nativeMetadata,
+    tokensBalances,
+    tokensMetadata,
+  } = useAssets();
+
+  return (
+    <>
+      <h2>All Tokens</h2>
+      <Tokens
+        getNativeBalance={getNativeBalance}
+        getNativeMetadata={getNativeMetadata}
+        getTokensBalances={getTokensBalances}
+        getTokensMetadata={getTokensMetadata}
+        nativeBalance={nativeBalance}
+        nativeMetadata={nativeMetadata}
+        tokensBalances={tokensBalances}
+        tokensMetadata={tokensMetadata}
+      />
```

Comment:
is there a reason to not use useAssets inside Tokens?
A good reason would be if some state from useAssets is reused in two or more components here, but I only see one.